apps/glusterfs/brick_create.go

```diff
@@ -10,35 +10,22 @@
 package glusterfs
 
 import (
+	"sync"
+
 	"github.com/heketi/heketi/executors"
 	wdb "github.com/heketi/heketi/pkg/db"
 	"github.com/heketi/heketi/pkg/utils"
 )
 
-type CreateType int
-
-const (
-	CREATOR_CREATE CreateType = iota
-	CREATOR_DESTROY
-)
-
-func createDestroyConcurrently(db wdb.RODB,
-	executor executors.Executor,
-	brick_entries []*BrickEntry,
-	create_type CreateType) error {
-
+func CreateBricks(db wdb.RODB, executor executors.Executor, brick_entries []*BrickEntry) error {
```

Comment:
Thank you for attacking this crazy function and working toward a sane interface! :-)